addon/components/o-s-s/select.hbs

```diff
@@ -17,37 +23,45 @@
     </div>
   </div>
 
-  {{#if this.displaySelect}}
-    <OSS::InfiniteSelect
-      @items={{@items}}
-      @onSearch={{this.onSearch}}
-      @onSelect={{this.onSelect}}
-      @searchEnabled={{this.searchEnabled}}
-      @searchPlaceholder={{this.searchPlaceholder}}
-      {{on-click-outside this.onClickOutside useCapture=true}}>
-      <:option as |item|>
-        <div class="item-wrapper fx-row fx-xalign-center fx-malign-space-between {{if (eq @value item) 'selected'}}">
-          {{#if (has-block "option")}}
-            {{yield item to="option"}}
-          {{/if}}
+  {{#in-element this.portalTarget insertBefore=null}}
+    {{#if this.displaySelect}}
+      <OSS::InfiniteSelect
+        @items={{@items}}
+        @onSearch={{this.onSearch}}
+        @onSelect={{this.onSelect}}
+        @searchEnabled={{this.searchEnabled}}
+        @searchPlaceholder={{this.searchPlaceholder}}
+        id={{this.portalId}}
+        class="margin-top-px-0"
+        {{on-click-outside this.onClickOutside}}
+        {{on "click" this.noop}}
```

Comment:
Do you really need this one?

Reply:
replied on the https://github.com/upfluence/oss-components/pull/366#discussion_r1514798546